verticaltabs.js

```diff
@@ -617,6 +636,40 @@ VerticalTabs.prototype = {
     }
   },
 
+  resizeTabs: function () {
+    if (this.resizeTimeout > 0) {
+      this.window.clearTimeout(this.resizeTimeout);
+      this.resizeTimeout = -1;
+    }
+    if (this.mouseInside) {
```

Comment:
it seems that `this.needsResize` is not actually necessary and `this.mouseInside` does the job for it. I tried removing all references of `this.needsResize` without any problems.